code/modules/events/event_container.dm

```diff
@@ -189,7 +190,8 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 		new /datum/event_meta/no_overmap(EVENT_LEVEL_MAJOR, "Electrical Storm",		/datum/event/electrical_storm, 		0,	list(ASSIGNMENT_ENGINEER = 10, ASSIGNMENT_JANITOR = 5)),
 		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Drone Revolution",				/datum/event/rogue_maint_drones,	0,	list(ASSIGNMENT_ENGINEER = 10,ASSIGNMENT_MEDICAL = 10,ASSIGNMENT_SECURITY = 10)),
 		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Spider Infestation",				/datum/event/spider_infestation, 	0,	list(ASSIGNMENT_SECURITY = 10), 1),
-		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Bluespace Drive Instability",	/datum/event/bsd_instability,		0,		null, 1),
+		new /datum/event_meta(EVENT_LEVEL_MAJOR, "Bluespace Drive Instability",		/datum/event/bsd_instability,		0,	null, 1),
+		new /datum/event_meta(EVENT_LEVEL_MAJOR, "DeepmaintRooms",					/datum/event/backrooms_deepmaint,	0,	null, 1),
```

Comment:
Again, seems weird to have this in both event categories where the only difference here is it'll only happens once. Is this an oversight?

Reply:
TBH I was copying BSD Instability above :^)